common/lib/client/auth.js

```diff
@@ -437,6 +437,11 @@ var Auth = (function() {
 			tokenRequest(tokenRequestOrDetails, function(err, tokenResponse, headers, unpacked) {
 				if(err) {
 					Logger.logAction(Logger.LOG_ERROR, 'Auth.requestToken()', 'token request API call returned error; err = ' + Utils.inspectError(err));
+					if(!(err && err.code)) {
```

Comment:
I find this logic odd to read, wouldn't `if(err && !err.code)` make it simpler to comprehend?

Reply:
since it's inside the `if(err)` check then surely we just need `if(!err.code)` ?

Reply:
good point